source/WeaponSpecial/Heal_test.gd

```diff
@@ -0,0 +1,7 @@
+extends "res://source/unit/WeaponSpecial.gd"
```

Comment:
You can do `extends WeaponSpecial` here.
It's better because it won't cause issues in case the WeaponSpecial.gd file gets moved.